frontend/src/app/commonComponents/CovidResultGuidance.tsx

```diff
@@ -112,51 +114,47 @@ const setCovidResultInfo = (
                 href={t("testResult.cdcLink")}
                 target="_blank"
                 rel="noopener noreferrer"
+                key="cdc-link-two"
               >
                 cdc.gov
               </a>,
             ]}
           />
         </>
       );
-    default:
+    case TEST_RESULTS.UNDETERMINED:
       return (
         <>
           <p>{t("testResult.notes.inconclusive.p0")}</p>
           {isPatientApp && <p>{t("testResult.notes.inconclusive.p1")}</p>}
         </>
       );
+    default:
+      return null;
   }
 };
 
-const CovidResultGuidance = (props: Props) => {
-  const result = props.result;
-  const isPatientApp = props.isPatientApp;
-  const needsHeading = props.needsHeading;
+interface CovidResultGuidanceProps {
+  result: TestResult;
+  isPatientApp: boolean;
+  needsHeading: boolean;
+}
+const CovidResultGuidance = ({
+  result,
+  isPatientApp,
+  needsHeading,
+}: CovidResultGuidanceProps) => {
   const { t } = useTranslation();
-  const covidGuidanceArray: any = [];
-
-  if (needsHeading) {
-    covidGuidanceArray.push(
-      <p className="text-bold sr-guidance-heading">
-        {t("testResult.notes.h1")}
-      </p>
-    );
-  }
 
-  switch (result) {
-    case "UNDETERMINED":
-      covidGuidanceArray.push(
-        setCovidResultInfo("UNDETERMINED", isPatientApp, t)
-      );
-      break;
-    case "NEGATIVE":
-      covidGuidanceArray.push(setCovidResultInfo("NEGATIVE", isPatientApp, t));
-      break;
-    case "POSITIVE":
-      covidGuidanceArray.push(setCovidResultInfo("POSITIVE", isPatientApp, t));
-      break;
-  }
-  return covidGuidanceArray;
+  return (
+    <>
+      {needsHeading && (
```

Comment:
I'm noticing this heading "For Covid-19:" doesn't get added in the patient app view when you have a positive covid and positive flu result.

**Main**
<img width="1497" alt="Screenshot 2023-05-04 at 15 34 13" src="https://user-images.githubusercontent.com/20211771/236324143-56c6a299-e257-4693-b94f-0d5965f44d0b.png">



**This branch**

<img width="1283" alt="Screenshot 2023-05-04 at 15 38 31" src="https://user-images.githubusercontent.com/20211771/236324001-3f6d2e0a-b84d-41d4-ad58-41d2a15bf690.png">

Reply:
checking 👀